package/toltec-bootstrap/entware-reenable

```diff
@@ -31,6 +34,42 @@ UNIT
     systemctl enable --now opt.mount 2> /dev/null
 }
 
+# Mount /opt/share/zoneinfo to /usr/share/zoneinfo
+timezone-mount() {
+    mkdir -p "$entware_path"/share/zoneinfo
+
+    if [[ ! -f $systemd_timezone_mount_path ]]; then
+        log "Updating timezone.mount location"
+
+        if systemctl -q is-enabled timezone.mount; then
+            # Remove path to old file if still symlinked
+            systemctl disable timezone.mount
+        fi
+    fi
+
+    # Create systemd mount unit to mount over /opt on reboot
+    cat > "$systemd_timezone_mount_path" << UNIT
+[Unit]
+Description=Bind mount /opt/share/zoneinfo over /usr/share/zoneinfo
+DefaultDependencies=no
+Conflicts=umount.target
+Before=local-fs.target umount.target
+After=opt.mount
+
+[Mount]
+What=/opt/share/zoneinfo
```

Comment:
Is the extra level of indirection really needed here? If not, we could also remove the `After=opt.mount` dependency above.

```suggestion
What=/home/root/.entware/share/zoneinfo
```

Reply:
Fair enough